src/main/java/net/atos/zac/app/zaken/model/RESTBesluitIntrekkenGegevens.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.app.zaken.model;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+public class RESTBesluitIntrekkenGegevens {
+
+    public UUID besluitUuid;
+
+    public LocalDate vervaldatum;
+
+    public String vervalreden;
```

Comment:
Moet deze dan niet van het type Vervalreden zijn?

Reply:
Nee, dan werkt het niet. Heb ik ook een tijdje mee geworsteld, maar uiteindelijk gevonden hoe dat elders gebeurt met enums. Voor de code completion is de enum aan de kant van het gebruik ook belangrijker.